packages/manager/modules/exchange/src/external-contact/external-contact.html

```diff
@@ -3,220 +3,117 @@
+                        data-disabled="!ctrl.services.exchangeStates.constructor.isOk($row)"
+                    >
                         <span
-                            class="input-group-addon"
-                            data-ng-if="!ctrl.filter"
-                        >
-                            <span
-                                class="fa fa-search"
-                                aria-hidden="true"
-                            ></span>
-                        </span>
-                    </div>
-                </div>
-            </form>
-
-            <div class="table-responsive">
-                <table class="table table-hover">
-                    <thead>
-                        <tr>
-                            <th
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_externalEmailAddress"
-                            ></th>
-                            <th
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_displayName"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_creationDate"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_hiddenFromGAL"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_state"
-                            ></th>
-                            <th class="min-width text-center"></th>
-                        </tr>
-                    </thead>
-
-                    <tbody data-ng-if="ctrl.contactsLoading">
-                        <tr>
-                            <td colspan="6" class="text-center">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                    </tbody>
-
-                    <tbody
-                        data-ng-if="!ctrl.contactsLoading && ctrl.contacts.list.results.length === 0"
+                            data-translate="exchange_tab_EXTERNAL_CONTACTS_table_actions_menu_modify"
+                        ></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item
+                        data-on-click="setAction('exchange/external-contact/remove/external-contact-remove', $row.externalEmailAddress)"
```

Comment:
In order to harmonize with component properties ? :)
```suggestion
                       on-click="setAction('exchange/external-contact/remove/external-contact-remove', $row.externalEmailAddress)"
```